blob-module/ios/BlobModule.swift

```diff
@@ -18,12 +24,20 @@ public class BlobModule: Module {
                 blob.type
             }
             
+            Property("options") { (blob: Blob) in
+                blob.options
+            }
+            
+            Property("blobParts") { (blob: Blob) in
+                blob.blobParts
+            }
+            
             Function("slice") { (blob: Blob, start: Int?, end: Int?, contentType: String?) in
                 blob.slice(start: start ?? 0, end: end ?? blob.blobParts.count, contentType: contentType ?? "")
```

Comment:
this doesn't seem correct? blobParts.count is not the total size of blob, so it's not a good fallback value